examples/assets/ui/custom.ron

```diff
@@ -0,0 +1,25 @@
+#![enable(implicit_some)]
+Custom(
+    Repeat(
```

Comment:
We're on the verge of being able to create a neat little "text shadow" effect here if we add color and z deltas for each repeat.

Reply:
oh right! and if you draw the text with the black color and a bit bigger under the original, you also get borders

Reply:
@jojolepro don't forget about letter spacing and font kerning, though, which might cause issues with this approach. Are advanced font rendering techniques like that even supported, yet?

Reply:
not currently